libvast/include/vast/concept/parseable/vast/data.hpp

```diff
@@ -74,6 +74,7 @@ struct data_parser : parser_base<data_parser> {
       | parsers::addr
       | parsers::number
       | parsers::tf
+      | parsers::boolean
```

Comment:
This is not mentioned in the PR.

Reply:
This was not supposed to be in here. I'll remove and push again. 😅